tasks/test.py

```diff
@@ -50,9 +69,10 @@ def ensure_bytes(s):
 @task
 def install_tools(ctx):
     """Install all Go tools for testing."""
-    with ctx.cd("internal/tools"):
-        for tool in TOOL_LIST:
-            ctx.run("go install {}".format(tool))
+    with environ({'GO111MODULE': 'on'}):
```

Comment:
Why do we need this? I think this should be done on a separate PR, it does not look related to the rest of changes in this PR